packages/core/src/components/progress-indicator/progress-indicator.tsx

```diff
@@ -36,7 +36,7 @@ export class Progressindicator {
               {`
                 .spinner { animation: rotator 8s linear infinite; transform-origin: center; }
                 @keyframes rotator { 0% { transform: rotate(0deg); } 100% { transform: rotate(360deg); } }
-                .path { stroke-dasharray: 265; stroke-dashoffset: 0; transform-origin: center; stroke: #39870c; animation: dash 2s ease-in-out infinite; }
+                .path { stroke-dasharray: var(--_progress-indicator-spinner-stroke-dasharray, 265); stroke-dashoffset: 0; transform-origin: center; stroke: #39870c; animation: dash 2s ease-in-out infinite; }
```

Comment:
cc @hansgrimm @anneke: Ik heb iets nieuws bedacht, private custom properties.

Ik moet vanuit mijn tests voor Percy de animaties uitzetten. Het blijkt zeer complex te zijn om de inner workings van een component te wijzigen **en** deze wijzigingen bij Percy te krijgen.

Lang leve custom properties: Deze kan ik vanuit mijn test zetten, Percy neemt die keurig over. Maar ik wil absoluut niet dat dit een onderdeel van de public api wordt. Om dit toch enigszins expliciet te maken heb ik een custom property met `_` gemaakt.

Dit is al een conventie in de scriptwereld, ik heb 'm nu ook ingezet in onze CSS.